### ISSUE TITLE
Add clamp CSS function in modules as an option. 

### ISSUE THREAD

**Issue by kbysfm** [beaverbuilder/feature-requests#67]
**Describe the solution you'd like**
Include clamp options to control font size, margin, padding etc in modules for easier responsive / fluid layouts. 

**Describe alternatives you've considered**
The simple option would be to use a child theme and add CSS instead. It would be more accessible to the masses if it was baked into the GUI. 

**Additional context**
Great demonstration and basic concept of CSS clamp
https://youtu.be/dg488RrpNTc

**Comment by quasel** [comment]
Yeah some sort of fixed "with" in-between a min max would be nice to have - though I started to just a few lines for Flexbox/grid and do it myself ;) - there is an add-on that helps too - https://mrfent.com/beaver-builder-flexbox/

**Comment by justinledelson** [comment]
Any updates on this? Would love to have fluid typography.
